resources/arduino_files/camera_board/camera_board.ino

```diff
@@ -5,162 +5,261 @@
 #include "Adafruit_Sensor.h"
 #include "DHT.h"
 
-#define DHT_TYPE DHT22 // DHT 22  (AM2302)
-
-/* DECLARE PINS */
 const int DHT_PIN = 9;      // DHT Temp & Humidity Pin
-const int CAM_01_RELAY = 5;
-const int CAM_02_RELAY = 6;
+const int CAM_0_RELAY = 5;
+const int CAM_1_RELAY = 6;
 const int RESET_PIN = 12;
 
+// Utitlity Methods
+
+void toggle_led() {
+  digitalWrite(LED_BUILTIN, !digitalRead(LED_BUILTIN));
+}
+
+void turn_pin_on(int pin_num) {
+  digitalWrite(pin_num, HIGH);
+}
+
+void turn_pin_off(int pin_num) {
+  digitalWrite(pin_num, LOW);
+}
+
+// IO Handlers.
 
-/* CONSTANTS */
-Adafruit_MMA8451 accelerometer = Adafruit_MMA8451();
+class AccelerometerHandler {
+  public:
+    void init() {
+      ready_ = false;
+      has_reading_ = false;
+      ready();
+    }
+    void collect() {
+      if (ready()) {
+        accelerometer_.getEvent(&event_);
+        orientation_ = accelerometer_.getOrientation(); // Orientation
+        has_reading_ = true;
```

Comment:
`getEvent()` doesn't provide a status output, so we can't tell if something went wrong.

Do you mean can `has_reading_` be wrong?

Reply:
Yes, wondering if `getEvent()` could somehow fail and then `has_reading` would be outdated. Since `report()` just checks for `has_reading` it would just keep reporting the same values and we wouldn't be the wiser. Would be nice to reset `has_reading` to `false` each time somehow. Might not be necessary though.